javascript/javascript2/week3/warmup.js

```diff
@@ -0,0 +1,93 @@
+
+window.addEventListener("load", () => {
+  setTimeout(() => {
+    console.log("load efter 2,5 seconds");
+  }, 2500);
+});
+
+// function with delay parameter
+let delayfunction = (stringToLog,delay) =>{
+setTimeout(() => {
+    console.log(stringToLog)
+}, delay);
+}
+
+const btn1 = document.getElementById('btn1');
+btn1.onclick = () => {
+delayfunction("this string logget after 2 seconds of btn clicked   ", 2000)};
+
+
+// funtion with funtion parameter
+let earthLogger = () =>{
+    console.log("earth")
+}
+let saturenLogger = () =>{
+    console.log("saturen")
+}
+
+function planetLogFunction(callback){
+ callback();
+}
+planetLogFunction(saturenLogger);
+
+
+// find loction
+const demo = document.getElementById("demo");
+const locationBtn = document.getElementById("locBtn");
+locationBtn.onclick = function getLoction (){
```

Comment:
I would add an event listener here instead of assigning a function to the buttons `onclick` prop. In this simple example it works, however, if you're working on a team, on a big project, it can become extremely difficult to work with code where everything is being reassigned like this since it's not clear what the functionality is at any given time. 

I strongly recommend changing this to an event listener instead since it's a more idiomatic way of adding functionality to your html elements meaning that others will have an easier time understanding your code.